features/fixtures/rails3/app/app/controllers/breadcrumbs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class BreadcrumbsController < ActionController::Base
+  def initialize
```

Comment:
I don't normally see overriding of `#initialize` for a controller class. You should be able to use `Rails.cache` in your `#cache_read` method instead.